README.md

```diff
@@ -417,6 +418,10 @@ Edition compliant board equipped with a HiSilicon Kirin 620 SoC (8-core,
 
 To build for HiKey, please refer to [6. repo manifests](#6-repo-manifests).
 
+---
+### 4.9 Freescale MX6UL EVK
+TODO
```

Comment:
Please replace "TODO" with some of the description you have in the commit message of this platform.